routes/game.rb

```diff
@@ -80,9 +80,11 @@ class Api
 
                 game.save
               else
+                players = users.map { |u| [u.id, u.name] }.to_h
```

Comment:
i wonder will this break compatability with all pins?

Reply:
No, since this is in the non-pin case